src/pages/api/periodProduct.ts

```diff
@@ -2,26 +2,26 @@ import { NextApiRequest, NextApiResponse } from 'next';
 import { getDomain, getUuidFromCookie, redirectTo, redirectToError, setCookieOnResponseObject } from './apiUtils';
 import { isSessionValid } from './service/validator';
 import { PeriodProductType } from '../../interfaces';
-import { PERIOD_PRODUCT } from '../../constants';
+import { PERIOD_PRODUCT_COOKIE } from '../../constants';
 
 export const isCurrency = (periodPriceInput: string): boolean => {
     const regex = /^\d+(\.\d{1,2})?$/;
     return regex.test(periodPriceInput);
 };
 
-export const trimPeriodNameInput = (periodNameInput: string): string => {
-    return periodNameInput.trim().replace(/\s+/g, ' ');
+export const trimInputOfWhiteSpace = (input: string): string => {
```

Comment:
As mentioned, I'd move this and the `isCurrency` function to a shared utils file since they're both being used elsewhere now